configman/commandline.py

```diff
@@ -5,5 +5,9 @@
 # this will be expanded in the future when additional command line libraries
 # are supported
 
-import getopt as command_line
+try:
+    import argparse as command_line
+    from configman.def_sources.for_argparse import ArgumentParser
```

Comment:
Now any ImportError happening in for_argparse is going to be interpreted as if `argparse` isn't available.